flashplugin-x86_64.lkgv

```diff
@@ -0,0 +1,3 @@
+VERSION=23.0.0.185
+URL=https://fpdownload.adobe.com/pub/flashplayer/pdc/23.0.0.185/flash_player_ppapi_linux.x86_64.tar.gz
```

Comment:
This is not a shell file but a simple plain text file with KEY=VALUE pairs that I parse from the main script, so I'm not sure doing that would work.

I can however try to treat it as a shell script and source it directly instead of manually parsing it... I'll try that